refinery/config/settings/base.py

```diff
@@ -577,3 +578,5 @@ def get_setting(name, settings=local_settings):
 REFINERY_TUTORIAL_STEPS = refinery_tutorial_settings
 
 ANONYMOUS_USER_ID = -1
+
+TEST_RUNNER = "django.test.runner.DiscoverRunner"
```

Comment:
Existing configuration has `TEST_RUNNER = 'django_nose.NoseTestSuiteRunner`. Does this mean we don't need Nose and django-nose anymore?